deployments/overlays/nodeport/manifest.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: v1
```

Comment:
hmm is this file generated in some of the steps you were testing?

Reply:
Removed file